src/test/groovy/org/prebid/server/functional/BidderParamsSpec.groovy

```diff
@@ -273,4 +277,55 @@ class BidderParamsSpec extends BaseSpec {
         assert response.ext?.errors[ErrorType.GENERIC]*.code == [999]
         assert response.ext?.errors[ErrorType.GENERIC]*.message == ["no empty host accepted"]
     }
+
+    @PendingFeature
+    def "PBS should reject bidder when bidder params from request doesn't satisfy json-schema"() {
+        given: "BidRequest with bad bidder datatype"
+        def bidRequest = BidRequest.defaultBidRequest.tap {
+            imp[0].ext.prebid.bidder.generic.exampleProperty = PBSUtils.randomNumber
+        }
+
+        when: "PBS processes auction request"
+        def response = defaultPbsService.sendAuctionRequest(bidRequest)
+
+        then: "Response should not contain errors"
+        assert !response.ext?.errors
+
+        and: "Response should contain warnings"
+        assert response.ext?.warnings[ErrorType.PREBID]*.message ==
+                ["WARNING: request.imp[0].ext.prebid.generic was dropped with a reason: " +
+                         "request.imp[0].ext.prebid.bidder.generic failed validation",
+                 "WARNING: request.imp[0].ext must contain at least one valid bidder"]
+
+        and: "PBS should not call bidder"
+        assert bidder.getRequestCount(bidRequest.id) == 0
+
+        and: "seatbid should be empty"
+        assert response.seatbid.isEmpty()
+    }
+
+    @PendingFeature
+    def "PBS should not fail auction with storedAuctionResponse when request bidder params doesn't satisfy json-schema"() {
```

Comment:
Kinda pointless test. Stored auction response is for debugging purposes. Better check that the same logic is valid for AMP and stored requests.